engine/src/main/java/org/terasology/rendering/dag/nodes/DownSampleSceneAndUpdateExposureNode.java

```diff
@@ -206,10 +239,10 @@ private void downSampleSceneInto1x1pixelsBuffer() {
      * only if eye adaptation is enabled: an NPE would be thrown only in that case.
      */
     private void obtainStaticFBOs() {
-        downSampledScene[4] = frameBuffersManager.getFBO("scene16");
-        downSampledScene[3] = frameBuffersManager.getFBO("scene8");
-        downSampledScene[2] = frameBuffersManager.getFBO("scene4");
-        downSampledScene[1] = frameBuffersManager.getFBO("scene2");
-        downSampledScene[0] = frameBuffersManager.getFBO("scene1");
+        downSampledScene[4] = staticFBM.getFBO(SCENE_16_URN);
+        downSampledScene[3] = staticFBM.getFBO(SCENE_8_URN);
+        downSampledScene[2] = staticFBM.getFBO(SCENE_4_URN);
+        downSampledScene[1] = staticFBM.getFBO(SCENE_2_URN);
+        downSampledScene[0] = staticFBM.getFBO(SCENE_1_URN);
```

Comment:
I'd rename the method to createStaticFBOs(). And perhaps you could also rename createPBOs to createStaticPBOs()

By the way, does any other node need these? Because if the answer is no there is no point in publishing them on the staticFBM?